frontend/src/pages/Admin/ManageSheet/ManageSheet.tsx

```diff
@@ -211,7 +211,7 @@ const ManageSheet: React.FC<ManageSheetProps> = () => {
     { field: 'sectionName', headerName: 'Section', width: 250 },
   ]
 
-  if (loading === 'loading') return <p>...Loading</p>
+   if (loading === 'loading') return <LinearProgress />
```

Comment:
change that loading to 'isLoading' from query, and remove that 'loading' state from component :)